web/src/components/ThemeContainer.tsx

```diff
@@ -96,6 +96,11 @@ const createTheme = (
             },
           },
         },
+        MuiBreadcrumbs: {
+          defaultProps: {
+            dir: theme.palette.mode,
+          },
+        },
```

Comment:
❓ `theme.palette.mode` has possible values of `light` and `dark`, that seems wrong 🙈 Perhaps this fixes the issue by just supplying any (even invalid) values to overwrite the default. Please still use an actual valid value though (e.g. `theme.contentDirection`).

Reply:
I guess setting it to an invalid value removes the default 'auto' and instead falls back to the direction set somewhere else (being ltr/rtl).